### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by njam**
Acceptance test?

### CODE REVIEW

**Comment by njam** [comment]
@vogdb @tomaszdurka please review

File: tests/data/test-jquery.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<html>
+  <body>
+
+    <script type="text/javascript" src="js/jquery.js"></script
```
Comment (vogdb) [inline tests/data/test-jquery.html]: Closing bracket on </script ?

File: tests/source/WebNavigatorTests/TestCase.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace WebNavigatorTests;
+
+use Cocur\BackgroundProcess\BackgroundProcess;
+use WebNavigator\Navigator;
+
+class TestCase extends \PHPUnit_Framework_TestCase {
+
+    /** @var BackgroundProcess */
+    protected $_webServer;
```
Comment (vogdb) [inline tests/source/WebNavigatorTests/TestCase.php]: Couldn't find any usages of it. What is it for ?

File: tests/source/WebNavigatorTests/NavigatorTest.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    public function testGetHtml() {
+        $this->_navigator->get('/test1.html');
+        $this->assertSame('Hello <span>World</span>', $this->_navigator->getHtml('#id-hello'));
+    }
+
+    public function testIsDisplayed() {
+        $this->_navigator->get('/test1.html');
+        $this->assertSame(true, $this->_navigator->isDisplayed('#id-hello'));
```
Comment (vogdb) [inline tests/source/WebNavigatorTests/NavigatorTest.php]: Why not use assertTrue ?

**Comment by njam** [comment]
@vogdb did two adjustments!
Regarding `assertTrue`: I usually prefer `assertSame(true` and `assertSame(false` just because they align nicely and it's much easier to change it from `true` to `false` - double click the word and change it. wdyt?

**Comment by vogdb** [comment]
Oh. I see. Will use this style in the future.

File: tests/source/WebNavigatorTests/NavigatorTest.php [hunk 4]
```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace WebNavigatorTests;
+
+use WebNavigator\Navigator;
```
Comment (vogdb) [inline tests/source/WebNavigatorTests/NavigatorTest.php]: do we still need this `use` ?

**Comment by vogdb** [comment]
@njam the last comment remained only. The rest is lgtm.

**Comment by njam** [comment]
@vogdb please check last two commits

**Comment by vogdb** [comment]
very nice. lgtm.

**Pull request merged**